src/components/Card/Card.tsx

```diff
@@ -1,14 +1,14 @@
-import { CSSProperties, ReactChild } from "react";
+import { CSSProperties } from "react";
 import styled from "styled-components";
 
 export interface Props {
   onClick?: () => void;
-  children?: ReactChild | ReactChild[];
+  children?: React.ReactNode;
```

Comment:
`ReactChild` is now duplicate and not recommended anymore. Thus I replaced them with `React.ReactNode` which contains all possible JSX's elements as type:
https://github.com/typescript-cheatsheets/react#useful-react-prop-type-examples